.github/workflows/mutation-testing.yml

```
@@ -19,14 +19,21 @@ jobs:
+            ${{ runner.os }}-pnpm-
+
+      - name: Setup pnpm and install dependencies
+        run: |
+          corepack enable
+          corepack prepare $(node -p "require('./package.json').packageManager") --activate
+          pnpm config set store-dir ~/.pnpm-store
+          pnpm install --frozen-lockfile
 
       - name: Run mutation testing
         run: make test-mutation
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)